### DISCUSSION TITLE
The dreamshaper-8-inpainting model download failed, it stopped halfway and threw an error.

### DISCUSSION THREAD

**Discussion by yyyke** [intel/AI-Playground#73] [Q&A]
![1](https://github.com/user-attachments/assets/971c3d25-cd91-45ba-ae0e-4593d39f0210)
![屏幕截图(3)](https://github.com/user-attachments/assets/8528b152-03c8-4419-b8d4-c345e0c368e2)
![屏幕截图(4)](https://github.com/user-attachments/assets/eed95d35-bd20-40bc-94ad-79d7c8e0d432)

**Comment by Nuullll** [comment]
Have another try. It will resume the download.

**Comment by week-wyk** [comment]
i meet the same problem

**Comment by susantowijaya** [comment]
the developer need to give a feature to resume a broken download due to bad connection...
